Formula/mesa.rb

```diff
@@ -91,30 +91,28 @@ def install
 
     ENV.prepend_path "PATH", "#{venv_root}/bin"
 
-    mkdir "build" do
-      args = ["-Db_ndebug=true"]
-
-      if OS.linux?
-        args << "-Dplatforms=x11,wayland"
-        args << "-Dglx=auto"
-        args << "-Ddri3=true"
-        args << "-Dgallium-drivers=auto"
-        args << "-Dgallium-omx=disabled"
-        args << "-Degl=true"
-        args << "-Dgbm=true"
-        args << "-Dopengl=true"
-        args << "-Dgles1=enabled"
-        args << "-Dgles2=enabled"
-        args << "-Dgallium-xvmc=disabled"
-        args << "-Dvalgrind=false"
-        args << "-Dtools=drm-shim,etnaviv,freedreno,glsl,nir,nouveau,xvmc,lima"
-      end
-
-      system "meson", *std_meson_args, "..", *args
-      system "ninja"
-      system "ninja", "install"
+    args = std_meson_args + ["-Db_ndebug=true"]
```

Comment:
Not a fan of putting `std_meson_args` here, personally. Better to just include it separately in the call to `meson`.

Reply:
I don't mind about that fix, but it will trigger another 4.5h run on Linux. I promise to address it in #101859.